### PR TITLE
PgDayParis

### PR DESCRIPTION
**Pull request by rodo**

### CODE REVIEW

**Comment by rodo** [comment]
Need a review by a good english speaker

**Comment by marco44** [comment]
I'll read this …

File: content/pgday-paris-2018.md [hunk 1]
```diff
@@ -1,94 +1,94 @@
...
+Started in 2015 as a natural extension to the PostgreSQL User Group, Paris' Meetup community, pgDay Paris has continued to grow each year, bringing the very best of the PostgreSQL international community to the French capital. Many thanks to Vik and all the volunteers who make this event so well organized and pleasant.
+The speakers, this year, had the chance to speak in the beautiful Saint-Martin venue, in front of about 200 top notch PostgreSQL fans. The single speaker track contained 7 presentations of 50 minutes, a couple of lightning talks, and one introduction and conclusion from Vik Fearing. Some of us are now frequent attendees of this event which gets better every year!
 
 
 
 
-### [De-mystifying contributing to PostgreSQL](https://www.postgresql.eu/events/pgdayparis2018/schedule/session/1704-de-mystifying-contributing-to-postgresql/)
+### [De-mystifying contributing to PostgreSQL](https://www.PostgreSQL.eu/events/pgdayparis2018/schedule/session/1704-de-mystifying-contributing-to-postgresql/)
```
Comment (wo0dyn) [inline content/pgday-paris-2018.md]: Are you sure you wanted to replace `postgresql` by `PostgreSQL` even in URLs @marco44? 😉

**Comment by marco44** [comment]
One regex too far :)

File: content/pgday-paris-2018.md (hunk 1 above)
Comment (marco44) [reply content/pgday-paris-2018.md]: Solved

**Comment by rodo** [comment]
Thanks @wo0dyn

**Pull request merged**